modules/admin-ui/src/main/webapp/index.html

```diff
@@ -146,7 +146,6 @@
     <script src="scripts/shared/services/services.js"></script>
     <script src="scripts/shared/services/resourceHelperService.js"></script>
     <script src="scripts/shared/services/authService.js"></script>
-    <script src="scripts/shared/services/cryptService.js"></script>
```

Comment:
The cryptservice is also included in login-html, would you mind removing it there as well?

Reply:
Done.